hera_sim/__init__.py

```diff
@@ -2,36 +2,43 @@
 from pathlib import Path
 
 try:
-    from importlib.metadata import version, PackageNotFoundError
+    from importlib.metadata import PackageNotFoundError, version
 except ImportError:
```

Comment:
Maybe want to add a `# pragma: nocover` here so codecov ignores this?

Reply:
I've added a global ignore of lines with `except ImportError` in the .coveragerc